### ISSUE TITLE
Save the binded tools 

### ISSUE THREAD

**Issue by Ziassan** [EngineHub/WorldEdit#1306]
**Is your feature request related to a problem? Please describe.**

When using a lot of different brushs/tools, including "complex" ones like % based spheres with masks and all, you have to redo every single tool every time you load the World.

**Describe the solution you'd like**

It would be very helpful to keep the tools and brushs saved, or allow a way to save the toolbar for next time. Would really made working on big projects easier.

**Describe alternatives you've considered**

I've tried saving every command in a text file and binding them one by one, but it's still a chore and keeping it updated can be annoying.

Thanks for your time.

**Comment by MattBDev** [comment]
We are about to start work on a brush specification in FAWE IntellectualSites/FastAsyncWorldEdit#692. Have you guys started any work on this or any general thoughts behind it?

**Comment by octylFractal** [comment]
No, not yet. I imagine it'll be part of the general brush / tool overhaul that @me4502 is planning to do at some point, probably as a WE 8 feature.

**Comment by MattBDev** [comment]
Are you fine if we start work on this? The plan was to contribute all this back to upstream anyway.

**Comment by octylFractal** [comment]
I'll defer to @me4502 on that, but I imagine it'd be fine. I'd like to look at/review a specification before you write too much code that you intend to upstream.

**Comment by kienerxyzzy** [comment]
Also (if possible) default the wand item (and really, any worldedit default wand) to something else. I play both creative and survival on my modpack, and I'm tired of wooden axes not chopping trees

**Comment by me4502** [comment]
In general we're unlikely to make it something else. Most items in MC have uses now. The wooden axe is so well known as the wand item at this point that changing it would hit more backlash than it's worth, and there's not really another good option.

Rather than having the default changed for everyone, you should either just not give yourself admin perms in your survival worlds, change the default in the config for your modpack, or instead just unbind the wand for yourself.

**Comment by kienerxyzzy** [comment]
ok
